siip.tex

```diff
@@ -78,48 +82,56 @@
           review one another's work in small chunks \cite{wilson_best_2014}, 
           and contribute back their own improvements to the main project.
+          Thus open-source projects can focus concentration on particularly needed tasks
+          even from diffuse effort.
 
-          So, why aren't professors sharing their lesson materials online, 
+          So why aren't professors sharing their lesson materials online, 
           collaborating on canonical lesson sets, diffing and merging similar 
-          lessons, and reviewing one another's learning materials? \textbf{Our 
-          goal is to explore the possibility that curriculum development for 
-          university courses can operate like open source software development 
-          does.}
-           
+          lessons, and reviewing one another's learning materials?
           Previous attempts at collaborative curriculum development have met with 
-          challenges. Most notable among these is Software Carpentry (with 
-          which the PI and Co-PI are quite involved) 
-          \cite{wilson_software_2014}.  Software Carpentry teaches 
+          challenges, including instructor inertia, an unclear process for collaboration,
+          lack of a committed core team, differing visions, and particular institutional needs.
+          %Q:  Why?  If you raise the question, we need at least a stab at an answer here.
+          % I know for myself the fact that I work in triage is a major factor.
+          \textbf{Given our experience with successsful collaborative curriculum development
+          in niche areas, our goal is to explore the possibility that curriculum development for 
+          university courses can operate as well as open source software development 
+          does.}
+          
+          Most notable among successful efforts is Software Carpentry (a program
+          which the PI and Co-PI have been quite involved with) 
+          \cite{wilson_software_2014}.  Since 2011, the Software Carpentry program has taught 
           computational skills for scientists using a shared curriculum hosted 
-          on GitHub. Hundreds of instructors accross the planet use and remix 
-          the curriculum, but our experience showed that while these 
+          on GitHub. Hundreds of instructors across the planet use and remix 
+          the lesson materials, but our experience showed that while these 
           instructors used and improved on the original material, they rarely 
           contributed their changes back to the master material. Thus, versions 
           of the original material tend to diverge rather than converge 
-          \cite{wilson_software_2014,wilson_software_2014-1}. 
+          \cite{wilson_software_2014,wilson_software_2014-1}.
+          % TODO:  comment on how SWC has successfully surmounted the other challenges noted above
```

Comment:
by forcing all new instructors to make arbitrary and likely-to-be-rejected PRs to become instructors? I'm not sure this qualifies as  successful surmounting. Anyway, Greg certainly doesn't think so.